src/plan/services/plan.service.ts

```
@@ -7,6 +7,7 @@ import { UpdatePlanResponseDto } from 'src/plan/dtos/responses/update-plan.respo
 import { AvailablePlanEntity } from 'src/plan/entities/available-plan.entity';
 import { PlanHistoryEntity } from 'src/plan/entities/plan-history.entity';
 import { PlanEntity } from 'src/plan/entities/plan.entity';
+import { PlanHistoryNotExist } from 'src/plan/exceptions/plan-history-not-exist';
```

ESLint

[error] 'PlanHistoryNotExist' is defined but never used.

(@typescript-eslint/no-unused-vars)